### PR TITLE
sig: add description for expression SIG

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

**Comment by zz-jason** [comment]
@gingerkidney @qw4990 PTAL

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Join
+
+The minimal requirement for contributors to join the Expr SIG is to have at
+least the **Active Contributor** community role in the expression package.
+
+### From Contributor to Active Contributor
+
+Requirement:
+
+* Have at least 8 PRs get merged in the expression package within one yeay.
```
Comment (qw4990) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]: yeay/year

Comment (zz-jason) [reply special-interest-groups/sig-expr/roles-and-organization-management.md]: done

File: special-interest-groups/sig-expr/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Expression Special Interest Group(Expr SIG)
+
+Expression SIG covers the tidb expression implementations. We are focused on
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]:
```suggestion
Expression SIG covers TiDB expression implementations. We are focused on
```

File: special-interest-groups/sig-expr/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+the development and maintenance of the TiDB [Function and
+Operator](https://pingcap.com/docs/dev/reference/sql/functions-and-operators/reference/)
+behavior and their dependencies, including but not limited to:
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]:
```suggestion
, including but not limited to:
```

File: special-interest-groups/sig-expr/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* Support new features.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]:
```suggestion
* Supporting new features.
```

File: special-interest-groups/sig-expr/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Improve the performance.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]:
```suggestion
* Improving the performance.
```

File: special-interest-groups/sig-expr/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+* Fix the bugs.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]:
```suggestion
* Fixing bugs.
```

File: special-interest-groups/sig-expr/README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Wednesdays at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: [pingcap.zoom.us/j/3190482744](https://pingcap.zoom.us/j/3190482744)
+* Meeting Notes: [Meeting Notes and Agenda](https://docs.google.com/document/d/1bHZs-qj91jH0crUFgROQ0P076rosDQNm2CsUTvDr0zo/edit)
+
+## Contact
+
+* Slack: channel #sig-expr in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/README.md]: tidbcommunity - > TiDB Community?

File: special-interest-groups/sig-expr/member-list.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+# SIG Expr Member List
+
+This member list is updated bi-weekly.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/member-list.md]:
```suggestion
This member list is updated on a  bi-weekly basis.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
+# SIG Expr Roles and Organization Management
+
+## Join && Promotion
+
+## Join
+
+The minimal requirement for contributors to join the Expr SIG is to have at
+least the **Active Contributor** community role in the expression package.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]: > in the expression package

Can we say "in the expression package subproject"?

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+### From Contributor to Active Contributor
+
+Requirement:
+
+* Have at least 8 PRs get merged in the expression package within one year.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Have at least 8 PRs merged in the expression package subproject within one year.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+* Nominated by at least 2 Expr SIG Reviewers, Committer or Maintainer.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]: > 2 Expr SIG Reviewers, Committer or Maintainer.
 
1 Committer or 1 Maintainer?

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+
+### From Active Contributor to Reviewer
+
+Requirement:
+
+* Have at least 30 PRs get merged in the expression package within one year.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Have at least 30 PRs merged in the expression package subproject within one year.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+### From Contributor to Active Contributor
+
+Requirement:
+
+* Have at least 8 PRs get merged in the expression package within one year.
+* Nominated by at least 2 Expr SIG Reviewers, Committer or Maintainer.
+
+### From Active Contributor to Reviewer
+
+Requirement:
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
Requirements:
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+## Join && Promotion
+
+## Join
+
+The minimal requirement for contributors to join the Expr SIG is to have at
+least the **Active Contributor** community role in the expression package.
+
+### From Contributor to Active Contributor
+
+Requirement:
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
Requirements:
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+* Have at least 8 PRs get merged in the expression package within one year.
+* Nominated by at least 2 Expr SIG Reviewers, Committer or Maintainer.
+
+### From Active Contributor to Reviewer
+
+Requirement:
+
+* Have at least 30 PRs get merged in the expression package within one year.
+* Nominated by at least 2 Committer or Maintainer.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Nominated by at least 2 Committers or 2 Maintainers.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+
+### Active Contributor
+
+**Automatically retired**:
+
+* No contribution to the expression package within one year.
+
+**How to rejoin**:
+
+* Should meet the requirement of Active Contributor again.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Should meet the requirements for Active Contributor again.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+
+* Haven't reviewed or merged any PR in the expression package within 3 monthes.
+* Not appeared in the SIG meeting without any reason continuously 3 times.
+* At least 2 Committers or Maintainers thinks that Reviewer is not qualified.
+
+How to retain:
+
+* Reviewd or merged 5 PRs within 1 month
+
+## Voluntary Quit or Demotion
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
## Voluntarily Quit or Demote
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+
+Please email to [Yanqing Zhang](mailto:zhangyanqing@pingcap.com) if you are
+going to voluntaryly quit or demotion
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
going to quit or demote voluntarily.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 19]
```diff
@@ -0,0 +1,60 @@
...
+
+* No contribution to the expression package within one year.
+
+**How to rejoin**:
+
+* Should meet the requirement of Active Contributor again.
+
+### Reviewer
+
+Demoted to Active Contributor, satisifies at least 1 of the following
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
For a Reviewer to be demoted to an Active Contributor,  he/she must satisfy at least 1 of the following conditions:
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 20]
```diff
@@ -0,0 +1,60 @@
...
+scenarios:
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion:
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 21]
```diff
@@ -0,0 +1,60 @@
...
+
+* Haven't reviewed or merged any PR in the expression package within 3 monthes.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Haven't reviewed or merged any PR in the expression package subproject within 3 months.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 22]
```diff
@@ -0,0 +1,60 @@
...
+* Not appeared in the SIG meeting without any reason continuously 3 times.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Haven't appeared in the SIG meeting without any reason for 3 times continuously.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 23]
```diff
@@ -0,0 +1,60 @@
...
+* At least 2 Committers or Maintainers thinks that Reviewer is not qualified.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* At least 2 Committers or Maintainers agree that the Reviewer is not qualified.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 24]
```diff
@@ -0,0 +1,60 @@
...
+
+How to retain:
+
+* Reviewd or merged 5 PRs within 1 month
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Have reviewed or merged 5 PRs within 1 month
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 25]
```diff
@@ -0,0 +1,60 @@
...
+### Reviewer
+
+Demoted to Active Contributor, satisifies at least 1 of the following
+scenarios:
+
+* Haven't reviewed or merged any PR in the expression package within 3 monthes.
+* Not appeared in the SIG meeting without any reason continuously 3 times.
+* At least 2 Committers or Maintainers thinks that Reviewer is not qualified.
+
+How to retain:
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
How to retain the Reviewer role:
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 26]
```diff
@@ -0,0 +1,60 @@
+# SIG Expr Roles and Organization Management
+
+## Join && Promotion
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]: Why $$?

File: special-interest-groups/sig-expr/roles-and-organization-management.md (hunk 9 above)
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
the **Active Contributor** community role in the expression package.
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 27]
```diff
@@ -0,0 +1,60 @@
...
+
+## Join
+
+The minimal requirement for contributors to join the Expr SIG is to have at
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
The minimal requirement for contributors to join the Expr SIG is to have
```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 28]
```diff
@@ -0,0 +1,60 @@
...
+least the **Active Contributor** community role in the expression package.
+
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
## Promotion

```

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 29]
```diff
@@ -0,0 +1,60 @@
...
+* Have at least 30 PRs get merged in the expression package within one year.
+* Nominated by at least 2 Committer or Maintainer.
+
+### From Reviewer to Committer
+
+Requirement:
+
+* Have at least 60 PRs get merged in the expression package within one year.
+* Have reviewed at least 20 PRs in the expression package within one year.
+* Nominated by at least 2 Committer or Maintainer.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]: The same edits as above

File: special-interest-groups/sig-expr/README.md (hunk 7 above)
Comment (zz-jason) [reply special-interest-groups/sig-expr/README.md]: I prefer to keep the old one. It's also keep the same form of what we described in [Vectorized Expression Working Group](https://github.com/pingcap/community/blob/master/working-groups/wg-vec-expr.md)

File: special-interest-groups/sig-expr/roles-and-organization-management.md [hunk 30]
```diff
@@ -0,0 +1,64 @@
...
+* Have at least 30 PRs get merged in the expression package within one year.
+* Nominated by at least 2 Committers or 2 Maintainers.
+
+### From `Reviewer` to `Committer`
+
+Requirement:
+
+* Have at least 60 PRs get merged in the expression package within one year.
+* Have reviewed at least 20 PRs in the expression package within one year.
+* Nominated by at least 2 Committer or Maintainer.
```
Comment (dcalvin) [inline special-interest-groups/sig-expr/roles-and-organization-management.md]:
```suggestion
* Nominated by at least 2 Committers or Maintainers.
```

**Comment by dcalvin** [comment]
@zz-jason Seems some comments are not handled. PTAL again.

**Comment by zz-jason** [comment]
@dcalvin Comments addressed, PTAL.

**Pull request merged**